src/scripts/bundle_proxy_win.bat

```diff
@@ -25,10 +25,10 @@ mkdir %DISTWORDSPATH% > nul 2> nul
 mkdir %TARGETPACKAGESPATH% > nul 2> nul
 
 rem Getting all the requirements
-python -m pip install -r %ETH2REQUIREMENTSPATH% --target %TARGETPACKAGESPATH%
+python -m pip install -r %ETH2REQUIREMENTSPADISTBINPATHTH% --target %TARGETPACKAGESPATH%
```

Comment:
why such a long name `ETH2REQUIREMENTSPADISTBINPATHTH` ?

Reply:
I just copied what they had. Someone wanted to make this cryptic :) haha.

Reply:
I will check the original and minimize my changes to make unnecessary things drop out.

Reply:
Fixed.